src/Explorer/Tree/ResourceTreeAdapter.tsx

```diff
@@ -276,6 +276,8 @@ export class ResourceTreeAdapter implements ReactAdapter {
     if (
       userContext.apiType === "Mongo" &&
       this.container.isNotebookEnabled() &&
+      !userContext.databaseAccount.properties.isVirtualNetworkFilterEnabled &&
```

Comment:
>isVirtualNetworkFilterEnabled [](http://example.com/codeflow?start=46&length=29)

you might need another property for private endpoints..

Reply:
Still figuring out best way to figure that out. Will send a follow up PR with that change when it's ready.